LibertyDSNP/example-client
# Feat/filtering hashtags

Purpose
---------------

Solution
---------------

Change summary
---------------
* A detailed list of bulleted

Steps to Verify
----------------
Run populate script and check that only #foodee tagged things are showing

Additional details / screenshot
----------------
- Any supplemental pictures or material
- ![Screenshot]()
